devel/fossil/Portfile

```diff
@@ -67,5 +68,5 @@ not harm the integrity of a repository.
 "
 
 livecheck.type      regex
-livecheck.url       ${homepage}fossil/uv/download.html
-livecheck.regex     {Version ([\d.]*)}
+livecheck.url       ${homepage}home/juvlist
```

Comment:
Is this really an improvement over using the html page?

Reply:
the html page doesn't work anymore and you need to parse the json

Reply:
I successfully looked at the HTML page. It also seems to have releases detailed on it. Why doesn't it work?

Reply:
Have you used `curl` or the browser? because the content is js rendered.